src/main/java/org/ballerinalang/command/cmd/ListCommand.java

```diff
@@ -51,6 +51,9 @@
     @CommandLine.Option(names = {"--help", "-h", "?"}, hidden = true)
     private boolean helpFlag;
 
+    @CommandLine.Option(names = {"--all", "-a"}, hidden = true)
+    private boolean allFlag;
```

Comment:
Lets include this in the help file [like this ](https://github.com/ballerina-platform/ballerina-update-tool/blob/master/src/main/resources/cli-help/ballerina-dist-remove.help#L7)
 https://github.com/ballerina-platform/ballerina-update-tool/blob/master/src/main/resources/cli-help/ballerina-dist-list.help